src/fu-util.c

```diff
@@ -2229,6 +2270,39 @@ fu_util_check_polkit_actions (GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_parse_filter_flags (FuUtilPrivate *priv, const gchar *filter, GError **error)
```

Comment:
In order to allow code-reuse for `fwupdtool`, what do you think about putting this in `fu-util-common.c` and rather than passing a `FuUtilPrivate` pass a pointer to `priv->filter_exclude` and `priv->filter_include`

Reply:
Works for me, you up for that or want me to do it?

Reply:
I can later on, but if you are eager go ahead :)